src/commands/base-command.ts

```diff
@@ -557,6 +561,9 @@ export class CommandHelper {
       return `\${{ ${node.id}.id }}`;
     } else if (answer === 'none') {
       return undefined;
+    } else if (answer === undefined && data[property.name]) {
+      console.log(`Invalid value for ${property.name}: ${data[property.name]}`);
```

Comment:
Why not just throw an error?

Reply:
I did that originally, but then I decided to copy what we do for accounts if you pass in an invalid valid:
```
 if (!account) {
    console.error(`Account ${selected_account} not found`);
    Deno.exit(1);
}
```

I don't necessarily have a strong preference either way, raising will also produce a non-zero exit code for CI flows - if you prefer raising though I can swap them both so it's consistent

Reply:
Thinking about it more we should probably throw more often and avoid `Deno.exit` when at all possible.

Reply:
Makes sense, I'll update these cases and see where else we're doing it (and either fix or make a ticket to fix the other places depending on how many)